### ISSUE TITLE
redundantFileprivate incorrectly applied

### ISSUE THREAD

**Issue by kmahar** [nicklockwood/SwiftFormat#493]
This seems similar to #478 and some other fileprivate issues.

I'm using Swift 5.1, swiftformat 0.40.13.

Running `swiftformat --swiftversion 5.1 test.swift` where `test.swift` is:
```swift
class Foo {
    fileprivate init() {}
}

class Bar: Foo {
    public override init() {
        super.init()
    }
}
```

changes the definition of `Foo` to:
```swift
class Foo {
    private init() {}
}
```

Which does not compile since the subclass can't access the initializer anymore.

**Comment by nicklockwood** [comment]
@kmahar fixed in 0.40.14, thanks for reporting it 👍

**Comment by kmahar** [comment]
Awesome, thank you!
